en/reference/query-language-reference.html

```diff
@@ -1232,7 +1232,9 @@ <h2 id="order-by">order by</h2>
 <p>
   The <a href="../ranking.html">rank profile</a> determines the rank score each document will get.
   Results are ordered by that value by default, but <code>order by</code> overrides that ordering.
-  Vespa does not optimize away the rank score computation in this case, it is still executed, even if the model score is thrown away. Use the built-in rank-profile <em>unranked</em> for optimal performance of sorting queries.
+  Vespa does not optimize away the rank score computation in this case, it is still executed, even if the model score is thrown away. Use the built-in rank-profile <em>unranked</em> for optimal performance of sorting queries.<br>
```

Comment:
Maybe clarify that the secondary is a tie-breaker if the relevance is equal?
```suggestion
  Vespa does not optimize away the rank score computation in this case, it is still executed, even if the model score is thrown away. Use the built-in rank-profile <em>unranked</em> for optimal performance of sorting queries.
  </p>
  <p>
```